cves/kernel/CVE-2014-3183.yml

```diff
@@ -278,8 +328,10 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
+  answer: false
   note:
+    The bug does not stem from the use of IPC methods.
```

Comment:
You might want to add details as to how you came to this conclusion. For example, does the function that this vulnerability occurs in access resources that are shared with other processes?

Reply:
I revisited the IPC section and this did involve IPC, so I fixed it and add an explanation.